CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/AppShell.xaml

```diff
@@ -24,9 +25,10 @@
             <Style Class="FlyoutItemLabelStyle" TargetType="Label">
                 <Setter Property="TextColor" Value="White" />
             </Style>
-            <Style ApplyToDerivedTypes="True"
-                   Class="FlyoutItemLabelStyle"
-                   TargetType="Layout">
+            <Style
+                ApplyToDerivedTypes="True"
+                Class="FlyoutItemLabelStyle"
+                TargetType="Layout">
```

Comment:
Update your formatter, we should all use the same formatting options, otherwise each change would be showing unnecessary changes in formatting which would be distracting the reviewer.